src/components/OverlayTooltip/InfoTooltip/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import OverlayTooltip from '../'
+import { InfoCircleIcon } from '@patternfly/react-icons'
+import style from './style.css'
+
+const InfoToolTip = ({ id, tooltip, placement, ...rest }) => (
```

Comment:
We loose some functionality when replacing FieldLevelHelp:
1. keyboard support
2. accessibility support

It seems that PF4 Tool tip seems a better replacement in those aspects (see [1]) 
Alternatively we can follow the guideline (see [2]) and re-implement this ourselves. 

[1] https://www.patternfly.org/v4/documentation/react/components/tooltip#on-icon
[2] https://www.patternfly.org/v4/get-started/accessibility-guide

Reply:
second thought on that: re-implementing is not worth the effort.
I would rather postpone this PR and implement it when we will start to use PF4. Then we could just use Tooltip component.
Alternatively if we are OK with losing functionality then let's make our wrapper easy to be replaced by PF4 component (i.e. the same props, similar usage pattern). The wrapper then will be just a temporary solution.

Reply:
@rszwajko @sgratch @sjd78 So are we OK with (hopefully temporarily) losing functionality using this approach?